DxDispatch/src/dxdispatch/DmlSerializedGraphDispatchable.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+class DmlSerializedGraphDispatchable : public Dispatchable
```

Comment:
We should update Guide.md with notes on how to use this. Also, how does one produce a flatbuffer file to begin with? This is a public tool, so it's a bit odd if we don't explain how to generate the files.